src/pages/ShowPage/useShowPage.ts

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+function useShowPage() {
+  const [showData, setShowData] = useState(false);
+
+  const handleButtonClick = () => {
+    setShowData(true);
+  };
+
+  const [latitude, setLatitude] = useState<number | null>(null);
+  const [longitude, setLongitude] = useState<number | null>(null);
+  const [address, setAddress] = useState<string | null>(null);
+  const [data, setData] = useState<string | null>(null);
```

Comment:
initialize with an empty string instead for both address n data

Reply:
Done